lib/version.js

```diff
@@ -1,7 +1,8 @@
 var Project = require('./project');
 var semver = require('semver');
 var _ = require('lodash');
-
```

Comment:
We generally leave a blank line between the block of `require()` imports and the rest of the file.